### ISSUE TITLE
EnvironmentModules `modulecmd.tcl` not found on Debian 10

### ISSUE THREAD

**Issue by zao** [easybuilders/easybuild-framework#3364]
When using `EnvironmentModules` (v4) as module-tool `easybuild/tools/modules.py` assumes that `modulecmd.tcl` is nested under `$MODULESHOME`.

On Debian 10, this is not the case and it lives in `/usr/lib/modulecmd.tcl` (symlink to `/usr/lib/x86_64-linux-gnu/modulecmd.tcl`), indicated by `$MODULES_CMD`.

This completely blocks the installation of 4.2.1 via the bootstrap script, my speculative patch against `develop` is as follows:

```diff
diff --git a/easybuild/tools/modules.py b/easybuild/tools/modules.py
index dda1eb40f..54e46fefa 100644
--- a/easybuild/tools/modules.py
+++ b/easybuild/tools/modules.py
@@ -1260,6 +1260,8 @@ class EnvironmentModules(EnvironmentModulesTcl):
     """Interface to environment modules 4.0+"""
     NAME = "Environment Modules v4"
     COMMAND = os.path.join(os.getenv('MODULESHOME', 'MODULESHOME_NOT_DEFINED'), 'libexec', 'modulecmd.tcl')
+    if not os.path.exists(COMMAND):
+        COMMAND = os.getenv('MODULES_CMD', 'MODULES_CMD_NOT_DEFINED')
     REQ_VERSION = '4.0.0'
     MAX_VERSION = None
     VERSION_REGEXP = r'^Modules\s+Release\s+(?P<version>\d\S*)\s'
```

**Comment by xdelaruelle** [comment]
I think #3816 has fixed this issue as described on #3815.

**Comment by boegel** [comment]
fixed with #3816, so closing, thanks for pointing that out @xdelaruelle!
